apps/charterafrica/src/pages/api/sourceAfrica/index.page.js

```diff
@@ -0,0 +1,13 @@
+const baseURL = "https://dc.sourceafrica.net/api";
```

Comment:
1. Lets put all Next.js apis into `/api/v1` folder. `/api is shared between Next.js and Payload CMS`. Without making this explicit decision, it will difficult down the line to figure out where a response comes from.
2. Lets match the navigation; hence this file should be in `opportunities/consultation/`
3. Lets call it `documents.js` i.e. `/api/opportunities/consultation/documents.js`.